HTAN_schema.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+
+import pandas as pd
+
+from schema_explorer import SchemaExplorer
+
+from csv_2_schemaorg import create_schema_classes 
+
+
+# path to base schema
+base_schema_path = "./data/schema_org_schemas/biothings_schema.jsonld"
```

Comment:
Change path to `./data/schema_org_schemas/biothings.jsonld` or any other json-ld file we decide rename the file.

Reply:
> Not sure if we want to put `csv_2_schemaorg` module into a package within the `utils` directory/sub-package like `ingresspipe.utils.csv_utils`?

That's a good idea - I'll add it there.

> Also need to update `./data/schema_org_schemas/HTAN.csv` to include a blank `Validation Rules` column.

Will do. 

> Suggestion: let's create a separate directory within `./data/` that contains the csv files to use (test/custom) say, `./data/csv` and also create a config parameter/object to allow configuration of the path to this. The objective of this would be to try and avoid as much hard-coding within the modules as possible.

Will handle that too.